### ISSUE TITLE
Error: HTTP Error 403: Forbidden Requesting returnOpenLoanOffers.  

### ISSUE THREAD

**Issue by combstraight** [BitBotFactory/poloniexlendingbot#410]
Hello, getting this error when opening lendingbot.py:

2017-06-26 18:55:08 Error: HTTP Error 403: Forbidden Requesting returnOpenLoanOffers.  Poloniex reports: '<!DOCTYPE html>

**Comment by rnevet** [comment]
check you KEY/SECRET, make sure you haeve IP filtering set correctly

**Comment by combstraight** [comment]
I have checked KEY/SECRET, they are both correct.  IP filtering has been set off.  I am sitll getting the same error.

**Comment by rnevet** [comment]
Try creating a new Key/Secret.

**Comment by tamaskan** [comment]
The "Nonce Error"-Section from https://www.poloniexlendingbot.com/docs/states-and-errors could be included in the Wiki

**Comment by rnevet** [comment]
https://github.com/BitBotFactory/poloniexlendingbot/blob/master/lendingbot.py#L96 
Nonce should be handled here.

**Comment by tamaskan** [comment]
yes it should. would be interesting if it works now and if he message appeared in his log

**Comment by dt2017** [comment]
This Nonce issue has been plaguing me for a while, Yesterday I done a git pull and setup a brand new key, but still getting Nonce issues.
If I restart the service then all good and it carries on, but for how long.. 

Any ideas?

**Comment by rnevet** [comment]
Basically if you are using the bot only once with it's own KEY/Secret you shouldn't have any noance issues... 

The noance is an increasing number that should always go up between API calls on an API key, a single bot handles this without issues.

**Comment by combstraight** [comment]
I actually found the issue.  I was logging in from a VPN because I was in China and Poloniex required my computer to complete a Captcha test for the API request.  The bot has been working fine in Hong Kong where no VPN is required.

**Comment by rnevet** [comment]
"Poloniex required my computer to complete a Captcha test for the API request" what a dump thing, can you send the response you received, maybe we can somehow indicate to the user. Did you manage to resolve it?

**Comment by arruah** [comment]
I have the same problem.

**Comment by rnevet** [comment]
@arruah follow the thread, not a bot issue.

**Comment by vensilver** [comment]
this is a captcha check error,

polonex is asking fill the captcha 

2017-07-21 15:30:16 Error: HTTP Error 403: Forbidden Requesting returnActiveLoans.  Poloniex reports: '<!DOCTYPE html>
<html lang="en">
<head>
        <title>Poloniex - Bitcoin/Cryptocurrency Exchange</title>
        <meta charset="UTF-8" />
        <style>a,body{color:#323A3C}body,h1{padding:0}.footer,.wrapper{text-align:center}body{margin:0;background-color:#04272A;font-family:HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;font-size:18px}a:hover{color:#A57211}h1{text-transform:uppercase;font-size:36px;margin-top:0;font-weight:400}.fine,.footer{font-size:.7em}#logo{min-width:100px;max-width:230px;margin:35px 10px}#content{border-bottom:3px solid #ffab06;background-color:#e5f0e6;padding:40px 10px}.wrapper{background-color:#fff}.fine{margin:22px 0 0}.footer{color:#fff;margin:40px 0}.footer a,.footer a:hover,.footer a:visited{color:#fff;display:inline-block}@media only screen and (max-width:350px){body{font-size:.9em}h1{font-size:1.6em}.fine{font-size:.75em}}p.cf-msg{margin:4px 0}#recaptcha_widget div div{width:auto!important;height:auto!important}</style>
</head>
<body>


how to solve this error

**Comment by tamaskan** [comment]
https://bitcointalk.org/index.php?topic=2026472.0 might be the key

**Comment by mchl18** [comment]
does the nonce have to be sequential? else, one could just add nonce+math.random() to achieve a higher nonce, or am i wrong. not sure if would help.
But also, concerning the bitcointalk discussion, we just had a PR that handled more cloudflare errors, but i think 403 was not one of them.

**Comment by rnevet** [comment]
Nonce most be sequential.

**Comment by lodha13** [comment]
Even I get this error urllib.error.HTTPError: HTTP Error 403: Forbidden

I am using the below code exactly and using the correct key have checked it multiple times.
https://pastebin.com/fbkheaRb

Please any help will be really appreciated.

**Comment by iyvinjose** [comment]
Error HTTP Error 403: Forbidden Requesting returnActiveLoans.  Poloniex reports:
I am also getting the same issue. Did anyone find a way to fix this?
